Orbitersdk/samples/ProjectApollo/src_sys/PanelSDK/Internals/Esystems.cpp

```diff
@@ -387,11 +409,14 @@ void FCell::UpdateFlow(double dt)
 	}
 
 	//Idle power load to prevent the fuel cells from stopping and cooling too much (has to be fixed at some point)
-	if (power_load < 160.0)
-		power_load = 160.0;
+	//if (power_load < 160.0)
```

Comment:
Personally I like to not leave commented out code as it takes away focus from the actual stuff going on. If it turns out it is needed again it can always be recovered from the git history. Do you agree with this?

Reply:
I agree. Bad practice from pre-git days.